src/crate/crash/repl.py

```diff
@@ -186,16 +186,19 @@ def __call__(self, buffer):
         current_line = buffer.document.text
         cursor_position = buffer.document.cursor_position
 
+
         if self.last_changed and self.is_prefix(current_line.lower(), self.last_changed.lower()):
             diff = len(self.last_changed) - len(current_line)
             current_line = self.last_changed + current_line[diff:]
 
-        new_line = re.sub(self.KEYWORD_RE, self.keyword_replacer, current_line)
+        new_line = re.sub(self.KEYWORD_RE, self.keyword_replacer, \
```

Comment:
`\` not required when having function arguments separated by newlines
same below